loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
```

Comment:
Perhaps renaming the variables to something like src and dest would make this a little clearer?  (target/link?) For some reason I find the to/from not intuitive.